scripts/sanity_chk/harness.py

```diff
@@ -55,6 +59,8 @@ class Test(Harness):
     RUN_PASSED = "PROJECT EXECUTION SUCCESSFUL"
     RUN_FAILED = "PROJECT EXECUTION FAILED"
 
+    faults = ["Unknown Fatal Error", "MPU FAULT", "Kernel Panic", "Kernel OOPS"]
```

Comment:
I think we also need "BUS FAULT" to handle k64 mpu

Reply:
done